src/module_wrap.cc

```diff
@@ -516,6 +524,7 @@ void ModuleWrap::Initialize(Local<Object> target,
   env->SetProtoMethod(tpl, "link", Link);
   env->SetProtoMethod(tpl, "instantiate", Instantiate);
   env->SetProtoMethod(tpl, "evaluate", Evaluate);
+  env->SetProtoMethod(tpl, "namespace", Namespace);
```

Comment:
`getNamespace` & `GetNamespace`? At least if this is idempotent I think that’d be a better name

Reply:
it can throw if it isn't instantiated.